packages/web-components/fast-components/src/radio/radio.styles.ts

```diff
@@ -42,174 +38,155 @@ export const radioStyles: (
 ) => ElementStyles = (context: ElementDefinitionContext, definition: RadioOptions) =>
     css`
     ${display("inline-flex")} :host {
-        --input-size: calc((${heightNumber} / 2) + ${designUnit});
+        --input-size: calc((${heightNumber} / 2) + (${strokeWidth} * 2));
         align-items: center;
         outline: none;
         margin: calc(${designUnit} * 1px) 0;
-        /* Chromium likes to select label text or the default slot when
-            the radio is clicked. Maybe there is a better solution here? */
+        ${
+            /*
+             * Chromium likes to select label text or the default slot when
+             * the radio button is clicked. Maybe there is a better solution here?
+             */ ""
+        }
         user-select: none;
         position: relative;
         flex-direction: row;
         transition: all 0.2s ease-in-out;
     }
-
     .control {
         position: relative;
-        width: calc((${heightNumber} / 2 + ${designUnit}) * 1px);
-        height: calc((${heightNumber} / 2 + ${designUnit}) * 1px);
+        width: calc(var(--input-size) * 1px);
+        height: calc(var(--input-size) * 1px);
         box-sizing: border-box;
-        border-radius: 999px;
-        border: calc(${strokeWidth} * 1px) solid ${neutralStrokeRest};
+        border-radius: 50%;
+        border: calc(${strokeWidth} * 1px) solid ${strokeControlStrongRest};
         background: ${neutralFillInputRest};
         outline: none;
         cursor: pointer;
     }
-
+    :host(:enabled:hover) .control {
+        border-color: ${strokeControlStrongHover};
+    }
+    :host(:enabled:active) .control {
+        background: ${neutralFillInputActive};
+        border-color: ${strokeControlStrongActive};
+    }
+    :host(:${focusVisible}) .control {
+        border-color: ${strokeControlStrongFocus};
+        outline: calc(${focusStrokeWidth} * 1px) solid ${strokeControlStrongFocus};
```

Comment:
This should be the focus stroke color. Will likely both look the same, but all focus treatments should use the same focus-specific recipes.

Reply:
updated